sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpMessageConverter.cs

```diff
@@ -218,38 +223,41 @@ public virtual AmqpMessage SBMessageToAmqpMessage(ServiceBusMessage sbMessage)
             }
             else
             {
-                if (sbMessage.AmqpMessage.Properties.CreationTime.HasValue)
+                if (annotatedMessage.Properties.CreationTime.HasValue)
                 {
-                    amqpMessage.Properties.CreationTime = sbMessage.AmqpMessage.Properties.CreationTime.Value.UtcDateTime;
+                    amqpMessage.Properties.CreationTime = annotatedMessage.Properties.CreationTime.Value.UtcDateTime;
                 }
-                if (sbMessage.AmqpMessage.Properties.AbsoluteExpiryTime.HasValue)
+                if (annotatedMessage.Properties.AbsoluteExpiryTime.HasValue)
                 {
-                    amqpMessage.Properties.AbsoluteExpiryTime = sbMessage.AmqpMessage.Properties.AbsoluteExpiryTime.Value.UtcDateTime;
+                    amqpMessage.Properties.AbsoluteExpiryTime = annotatedMessage.Properties.AbsoluteExpiryTime.Value.UtcDateTime;
                 }
             }
 
             // message annotations
 
-            foreach (KeyValuePair<string, object> kvp in sbMessage.AmqpMessage.MessageAnnotations)
+            foreach (KeyValuePair<string, object> kvp in annotatedMessage.MessageAnnotations)
             {
                 switch (kvp.Key)
                 {
                     case AmqpMessageConstants.ScheduledEnqueueTimeUtcName:
-                        if ((sbMessage.ScheduledEnqueueTime != null) && sbMessage.ScheduledEnqueueTime > DateTimeOffset.MinValue)
+                        DateTimeOffset scheduledEnqueueTime = annotatedMessage.GetScheduledEnqueueTime();
+                        if (scheduledEnqueueTime != default && scheduledEnqueueTime > DateTimeOffset.MinValue)
```

Comment:
nit: I think the `scheduledEnqueueTime > DateTimeOffset.MinValue` is redundant here because `DateTimeOffset.MinValue == default(DateTimeOffset)`

Reply:
Yeah good call.